src/main/java/com/mongodb/jdbc/MongoListCollectionsResult.java

```diff
@@ -1,6 +1,16 @@
 package com.mongodb.jdbc;
 
 public class MongoListCollectionsResult {
+    public static final String TABLE = "table";
```

Comment:
In the spec I see the table keyword is usually uppercase, is there a reason why we are setting it as lowercase here?

Reply:
Sounds good, thanks for the explanation